.github/workflows/ci.yml

```diff
@@ -113,13 +113,39 @@ jobs:
       - name: "Check out repository code"
         uses: "actions/checkout@v2"
       - name: "Setup environment"
-        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+        uses: "networktocode/gh-action-setup-poetry-environment@v2"
       - name: "Copy credentials"
         run: "cp development/creds.example.env development/creds.env"
       - name: "Build Container"
         run: "poetry run invoke build"
       - name: "Run Tests"
-        run: "poetry run invoke unittest"
+        run: "poetry run invoke unittest --failfast --keepdb"
```

Comment:
Not sure what benefit `keepdb` provides here?

Reply:
nothing, it was left behind